report/sql_tasks/sql_script.py

```diff
@@ -60,6 +60,10 @@ def _parse(self):
                 self.template_vars
             )
 
+        if not sqlparse.format(script_text, strip_comments=True).strip():
```

Comment:
Tried a version of this with sqlparse.parse and it had lots of edge cases. This seems to work fine and it's a onerliner.